src/credentials/attestation_verifier/DeviceAttestationVerifier.cpp

```diff
@@ -77,6 +81,25 @@ DeviceAttestationVerifier * gDacVerifier = &gDefaultDACVerifier;
 
 } // namespace
 
+#ifdef MATTER_FILE_SYSTEM_ENABLED
+FileAttestationTrustStore::FileAttestationTrustStore(const char * paaTrustStorePath)
+{
+    uint32_t certificateLength = 0;
+    std::ifstream ifs;
+
+    ifs.open(paaTrustStorePath, std::ifstream::in);
+
+    while (ifs.read(reinterpret_cast<char *>(&certificateLength), sizeof(uint32_t)))
+    {
+        char newline;
+        ifs.read(reinterpret_cast<char *>(mDerCertsRaw[mNumCerts]), certificateLength);
+        mDerCerts[mNumCerts] = ByteSpan(mDerCertsRaw[mNumCerts], certificateLength);
+        ++mNumCerts;
+        ifs.read(&newline, sizeof(char));
+    }
+}
+#endif
```

Comment:
If we put this entire code in a different file (not `src/credentials/attestation_verifier/DeviceAttestationVerifier.cpp`, but rather `src/credentials/attestation_verifier/FileAttestationTrustStore.h`, then that file, which will ONLY be included by host software like tests or chip-tool, can have any dependency you want, including glob.h, fstream.h, etc